src/camera.c

```diff
@@ -23,12 +23,6 @@ void	ft_initiate_image(t_image *image)
 
 void	ft_initiate_camera(t_cam *cam)
 {
-	cam->look_from.x = -0.0;
```

Comment:
I deleted look_from and look_at since they get parsed. Else you would overwrite the parsed values here. So it should be fine if they also get deleted on your part?

Reply:
Ah, misunderstood then. Thought the two variables get omitted completely. But you still use them in your code to save  the parsed values of camera view point and orientation vector, right? In that case, yeah we can delete these assignments

Reply:
Yes